packages/airnode-node/src/handlers/start-coordinator.ts

```diff
@@ -5,100 +5,176 @@ import * as logger from '../logger';
 import * as providers from '../providers';
 import { reportHeartbeat } from '../reporting';
 import { hasNoActionableRequests } from '../requests/request';
-import * as state from '../coordinator/state';
+import * as coordinatorState from '../coordinator/state';
```

Comment:
I like both the `startCoordinator` and `randomString` refactors but I don't think they should have been part of this PR. Especially the `startCoordinator` one since all the other handlers use the previous pattern. It should have been imo changed globally.

Reply:
Agree and also disagree :D . It shouldn't have been done in this PR, but doing it globally seems like a bigger effort. But I agree we should do this at some point and "play" with the code (globally) to improve it.